src/test/scala/org/apache/spark/sql/delta/LogStoreSuite.scala

```diff
@@ -271,6 +271,18 @@ class HDFSLogStoreSuite extends LogStoreSuiteBase {
   protected def shouldUseRenameToWriteCheckpoint: Boolean = true
 }
 
+class GcsLogStoreSuite extends LogStoreSuiteBase {
+
+  override val logStoreClassName: String = classOf[GcsLogStore].getName
+
+  testHadoopConf(
+    expectedErrMsg = "No FileSystem for scheme: fake",
+    "fs.fake.impl" -> classOf[FakeFileSystem].getName,
+    "fs.fake.impl.disable.cache" -> "true")
+
+  protected def shouldUseRenameToWriteCheckpoint: Boolean = true
```

Comment:
Does this test pass?

Reply:
shouldUseRenameToWriteCheckpoint had to be changed to false after isPartialWriteVisible was modified to false